test/test_datapipe.py

```diff
@@ -589,19 +598,40 @@ def test_serializable(self):
         dp_skip_comparison = {dp.iter.FileLister, dp.iter.FileOpener, dp.iter.StreamReader, dp.iter.Shuffler}
         # These DataPipes produce multiple DataPipes as outputs and those should be compared
         dp_compare_children = {dp.iter.Demultiplexer, dp.iter.Forker}
+
         for dpipe, dp_args, dp_kwargs in picklable_datapipes:
             datapipe = dpipe(input_dp, *dp_args, **dp_kwargs)  # type: ignore[call-arg]
```

Comment:
Better move in into the if branch and initialize as dp1, dp2 = dpipe(...) in case of dp_skip_comparison, it is going to be less confusing this way.